.rubocop.yml

```diff
@@ -16,87 +16,53 @@ Rails:
 #################### Layout ##############################
 
 Layout/AlignHash:
-  Enabled: false
+  EnforcedHashRocketStyle: table
 
 Layout/AlignParameters:
-  Enabled: false
+  EnforcedStyle: 'with_fixed_indentation'
 
 Layout/CaseIndentation:
-  Enabled: true
   EnforcedStyle: 'end'
   IndentOneStep: true
 
-Layout/ClosingHeredocIndentation:
-  Enabled: true
-  Exclude:
-    - 'db/migrate/**/*'
-
-Layout/ClosingParenthesisIndentation:
-  Enabled: false
-
 Layout/EmptyLineAfterMagicComment:
   Enabled: false
 
-Layout/EmptyLineBetweenDefs:
-  Enabled: false
-
-Layout/EmptyLines:
-  Enabled: false
-
 Layout/EmptyLineAfterGuardClause:
   Enabled: false
 
-Layout/EmptyLinesAroundAccessModifier:
-  Enabled: true
-
-Layout/EmptyLinesAroundExceptionHandlingKeywords:
-  Enabled: true
-
 Layout/EndAlignment:
-  Enabled: true
   EnforcedStyleAlignWith: 'variable'
 
 Layout/EndOfLine:
-  Enabled: false
-
-Layout/FirstParameterIndentation:
-  Enabled: false
+  EnforcedStyle: lf
 
 Layout/IndentationWidth:
   Enabled: true
   Width: 2
 
 Layout/IndentArray:
-  Enabled: true
   EnforcedStyle: 'consistent'
 
 Layout/IndentHash:
-  Enabled: true
   EnforcedStyle: 'consistent'
 
 Layout/IndentHeredoc:
   Enabled: true
   Exclude:
     - 'db/migrate/**/*'
 
-Layout/MultilineHashBraceLayout:
-  Enabled: false
-
 Layout/MultilineMethodCallBraceLayout:
   Enabled: false
 
 Layout/MultilineMethodCallIndentation:
-  Enabled: false
-
-Layout/MultilineMethodDefinitionBraceLayout:
-  Enabled: false
+  EnforcedStyle: 'indented'
+  Exclude:
+    - 'app/models/post.rb'
```

Comment:
What's up with this exclusion?

Reply:
It had Opinions about the visible_to scope and I disagreed with them

Reply:
What were the opinions? This is the only thing I'm confused about, I think.

Reply:
```
app/models/post.rb:83:7: C: Layout/MultilineMethodCallIndentation: Use 2 (not 0) spaces for indenting an expression spanning multiple lines.
      .or(where(privacy: Concealable::REGISTERED))
      ^^^
app/models/post.rb:84:7: C: Layout/MultilineMethodCallIndentation: Use 2 (not 0) spaces for indenting an expression spanning multiple lines.
      .or(where(privacy: Concealable::ACCESS_LIST, user_id: user.id))
      ^^^
app/models/post.rb:85:7: C: Layout/MultilineMethodCallIndentation: Use 2 (not 0) spaces for indenting an expression spanning multiple lines.
      .or(where(privacy: Concealable::ACCESS_LIST, id: PostViewer.where(user_id: user.id).select(:post_id)))
      ^^^
app/models/post.rb:86:7: C: Layout/MultilineMethodCallIndentation: Use 2 (not 0) spaces for indenting an expression spanning multiple lines.
      .or(where(privacy: Concealable::PRIVATE, user_id: user.id))
      ^^^
```

Reply:
I think I agree with it here that these should be indented. After merging #1111, can we do this?

Reply:
FINE :P